src/cmd/manage_nsfs.js

```diff
@@ -252,7 +252,7 @@ async function delete_bucket(data, force) {
         }
         await native_fs_utils.folder_delete(bucket_temp_dir_path, fs_context_fs_backend, true);
         await config_fs.delete_bucket_config_file(data.name);
-        return { code: ManageCLIResponse.BucketDeleted, detail: '', event_arg: { bucket: data.name } };
+        return { code: ManageCLIResponse.BucketDeleted, detail: { name: data.name }, event_arg: { bucket: data.name } };
```

Comment:
are there more cases where we want to add detail?

Reply:
for now I have added this in delete_account() and delete_bucket() operation only. We can print the name of object in message by setting the name property in detail(for any operation) otherwise the set message would be printed

Reply:
please fix all the cases in this PR **or** open another gap for the rest of the cases so we will handle it in the future

Reply:
Sure, I will open gap if there are more operations.

Reply:
@romayalon I just verified other then `connection_management` there is no need add details explicitly. I have updated it. Thanks

For every operation we are printing the name of the object in the message(the code parsed the name automatically if `detail.name` is defined).